agent/security_helper.py

```diff
@@ -118,3 +120,87 @@ def selinux_status():
             selinux_mode = row[1].strip()
 
     return {'enabled': selinux_enabled, 'mode': selinux_mode}
+
+
+AUDITED_CONFIG_FILES = [
+    '/etc/passwd',
+    '/etc/shadow',
+    '/etc/group'
+]
+SSHD_CONFIG_PATH = '/etc/ssh/sshd_config'
+SSHD_CONFIG_DEFAULTS = {
```

Comment:
Pls check the default values with `man sshd_config`.
And, an interesting observation: looks like sshd doesn't support 'Protocol' config parameter at all now - it was mentioned in the previous versions' config and man page, but completely missing now.

Reply:
Checked, doesn't match the [old docs](https://linux.die.net/man/5/sshd_config). For instance the default for `PermitRootLogin` is now `prohibit-password` and as you said `Protocol` is removed.
@vpetersson what do we do with that?

Reply:
Well, i take it we need to support it still, but perhaps make it conditional based on the version? Unfortunately this makes it a bit more messy. What version did this change in? I would imagine OpenSSH to be very backward compatible.

Reply:
These defaults are the same in Stretch and Buster. The [internet man](https://linux.die.net/man/5/sshd_config) is dated 2013 so it may be relevant to Jessie. I'm now checking mans in Jessie.

Reply:
In Jessie the default value for Protocol is "2" and PermitRootLogin is "yes".
So... I guess we need conditional handing here.